acl_upgrade.php

```diff
@@ -480,7 +483,8 @@
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'amendment', 'Amendments (write,addonly optional)', 'write');
   //Insert the 'lab' object from the 'patients' section into the Physicians group write ACL (added in 5.0.1)
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'lab', 'Lab Results (write,addonly optional)', 'write');
-
+  //Insert the 'Multipledb' object from the 'admin' section into the Administrators group write ACL (added in 5.0.1)
+  updateAcl($admin_write, 'Administrators','admin', 'Administration', 'Multipledb', 'multipledb','write');
   //DONE with upgrading to this version
   $acl_version = $upgrade_acl;
 }
```

Comment:
do you mean something like this : 
  addObjectAcl('admin', 'Administration', 'Groups', 'groups');

?